charts/pega/templates/_helpers.tpl

```diff
@@ -480,4 +482,15 @@ servicePort: use-annotation
 {{- else -}}
 {{- add .failureThreshold 1 -}}
 {{- end -}}
+{{- end -}}
+
+{{- define "pegaImagePullRegistrySecrets" }}
```

Comment:
```suggestion
{{- define "imagePullSecrets" }}
```

Reply:
Updated name from pegaImagePullRegistrySecrets to imagePullSecrets